### PR TITLE
Add option for title between duplicable rows

### PR DESCRIPTION
**Pull request by MatApple**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

File: src/js/components/SchemaForm.js [hunk 1]
```diff
@@ -161,16 +171,31 @@ class SchemaForm extends mixin(StoreMixin) {
...
         </button>
       </div>
     );
+
+    if (!title) {
+      return deleteButton;
+    }
+
+    return (
+      <div className="form-row-element duplicable-row-title-wrapper">
```
Comment (kennyt) [inline src/js/components/SchemaForm.js]: this thing needs a key. like what we do on line 176

**Comment by kennyt** [comment]
after my last comment, :ship:

File: src/js/components/SchemaForm.js (hunk 1 above)
Comment (kennyt) [inline src/js/components/SchemaForm.js]: done

**Pull request merged**